geode-apis-compatible-with-redis/src/acceptanceTest/java/org/apache/geode/redis/internal/executor/pubsub/PubSubNativeRedisAcceptanceTest.java

```diff
@@ -15,30 +15,70 @@
 
 package org.apache.geode.redis.internal.executor.pubsub;
 
-import org.apache.logging.log4j.Logger;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import org.apache.commons.lang3.SystemUtils;
 import org.junit.AfterClass;
+import org.junit.BeforeClass;
 import org.junit.ClassRule;
 
 import org.apache.geode.NativeRedisTestRule;
-import org.apache.geode.logging.internal.log4j.api.LogService;
 
 public class PubSubNativeRedisAcceptanceTest extends AbstractPubSubIntegrationTest {
-
-  private static final Logger logger = LogService.getLogger();
+  private static long socketTimeWaitMsec = 90000;
 
   @ClassRule
   public static NativeRedisTestRule redis = new NativeRedisTestRule();
 
+  @BeforeClass
+  public static void runOnce() throws IOException {
+    if (SystemUtils.IS_OS_LINUX) {
+      try {
+        BufferedReader bufferedReader =
+            new BufferedReader(new FileReader("/proc/sys/net/ipv4/tcp_fin_timeout"));
```

Comment:
This might be better as a try-with-resources: `try(BufferedReader bufferedReader = new BufferedReader(new FileReader("/proc/sys/net/ipv4/tcp_fin_timeout"))) {`